app/scripts/router.js

```diff
@@ -290,7 +292,9 @@ function (
       $(document).on('click', 'a[href^="/"]', this.onAnchorClick.bind(this));
     },
 
-    onAnchorClick: function (event) {
+    onAnchorClick: function (event, environment) {
+      /*eslint complexity: [2, 8] */
+      environment = environment || new Environment(this.window);
```

Comment:
is it possible to initialize the environment in the initialization of router.js: https://github.com/TDA/fxa-content-server/blob/issue-2351-tos-pp-new-window/app/scripts/router.js#L125 and use it here ?